creator-cards/owain-williams.json

```diff
@@ -3,7 +3,7 @@
   "sectionTitle": null,
   "countryCode": "uk",
   "slogan": "",
-  "type": "CREATOR",
```

Comment:
oops, sorry for the confusion i might have caused. 

The creator here was correct, within the feeds aray you have "LinkTagRssType" this has to be either personal or dotnet.

Reply:
Yep!